blockbuster/abci/check_tx.go

```diff
@@ -55,15 +57,24 @@ type (
 	// BaseApp is an interface that allows us to call baseapp's CheckTx method
 	// as well as retrieve the latest committed state.
 	BaseApp interface {
-		// GetContextForFinalizeBlock is utilized to retrieve the latest committed state.
-		GetContextForFinalizeBlock(tx []byte) sdk.Context
+		// CommitMultiStore is utilized to retrieve the latest committed state.
+		CommitMultiStore() storetypes.CommitMultiStore
 
 		// CheckTx is baseapp's CheckTx method that checks the validity of a
 		// transaction.
 		CheckTx(req *cometabci.RequestCheckTx) (*cometabci.ResponseCheckTx, error)
 
 		// Logger is utilized to log errors.
 		Logger() log.Logger
+
```

Comment:
porting over the old way of getting latest commit state since the `GetFinalizeBlockContext` is borked. kek

Reply:
👍